etc/rules/0015-wazuh_rules.xml

```diff
@@ -1,63 +1,229 @@
 <!--
-  -  OSSEC rules
-  -  Author: Daniel Cid.
-  -  Updated by Wazuh, Inc.
+  -  Wazuh rules
+  -  Created by Wazuh, Inc.
```

Comment:
The same comment about the header from the other repository.

Reply:
This was reverted in order to avoid changing the rules in the core repository.